src/components/device/configurators/thermostat-mode.component

```diff
@@ -59,6 +59,21 @@ limitations under the License.
         </label>
         <div class="check"></div>
       </li>
+      <div class="configurator-header">
+        <h4 class="config-title">Choose A Fan Mode</h4>
+      </div>
+      {{#if device['web:therm:autoSupported']}}
```

Comment:
Either we should check to see if the thermostat has a fan mode (how??) or we should remove this block - I don't think the auto mode has any relation to whether the thermostat has a fan mode

Reply:
Removed the if block